### PR TITLE
[build] Fix TF on MX Neuron DLC

### PR DESCRIPTION
**Pull request by saimidu**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [x] (If applicable) I've documented below the tests I've run on the DLC image
- [x] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [x] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

#### EIA/NEURON Checklist
* When creating a PR:
- [x] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [x] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [x] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Fix tensorflow version installed on MX Neuron image

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aws-vrnatham — DISMISSED** [review]
The neuron changes look good.

File: test/dlc_tests/conftest.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def delete_ssh_keypair():
     else:
         # Using private AMI, the EBS volume size is reduced to 28GB as opposed to 50GB from public AMI. This leads to space issues on test instances
         # TODO: Revert the configuration once DLAMI is public
-        params["BlockDeviceMappings"] = [{"DeviceName": "/dev/sda1", "Ebs": {"VolumeSize": 50,}}]
+        params["BlockDeviceMappings"] = [{"DeviceName": "/dev/sda1", "Ebs": {"VolumeSize": 90,}}]
```
Comment (saimidu) [inline test/dlc_tests/conftest.py:195]: Necessary for Neuron inference test.

File: test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 MX_TELEMETRY_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "test_mx_dlc_telemetry_test")
 
 @pytest.mark.model("mxnet-resnet-neuron")
-@pytest.mark.parametrize("ec2_instance_ami", [test_utils.UBUNTU_18_BASE_DLAMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_ami", [test_utils.NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2], indirect=True)
```
Comment (saimidu) [inline test/dlc_tests/ec2/mxnet/inference/test_mxnet_inference.py:31]: Necessary for Neuron inference test.

File: test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
 
 @pytest.mark.model("resnet")
-@pytest.mark.parametrize("ec2_instance_ami", [test_utils.UBUNTU_18_BASE_DLAMI_US_WEST_2], indirect=True)
+@pytest.mark.parametrize("ec2_instance_ami", [test_utils.NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2], indirect=True)
```
Comment (saimidu) [inline test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py:28]: Necessary for Neuron inference test.

File: test/dlc_tests/sanity/test_safety_check.py [hunk 4]
```diff
@@ -128,10 +148,10 @@ def _get_latest_package_version(package):
...
 @pytest.mark.canary("Run safety tests regularly on production images")
 @pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
-@pytest.mark.skipif(not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
-                    reason="Skipping the test to decrease the number of calls to the Safety Check DB. "
-                           "Test will be executed in the 'mainline' pipeline and canaries pipeline."
-                    )
+# @pytest.mark.skipif(not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
+#                     reason="Skipping the test to decrease the number of calls to the Safety Check DB. "
+#                            "Test will be executed in the 'mainline' pipeline and canaries pipeline."
+#                     )
```
Comment (saimidu) [inline test/dlc_tests/sanity/test_safety_check.py:154]: To be reverted.

File: src/config/build_config.py [hunk 5]
```diff
@@ -3,11 +3,11 @@
 ENABLE_EI_MODE = False
 # Please only set it to True if you are preparing an NEURON related PR
 # Do remember to revert it back to False before merging any PR (including NEURON dedicated PR)
-ENABLE_NEURON_MODE = False
+ENABLE_NEURON_MODE = True
```
Comment (saimidu) [inline src/config/build_config.py:6]: To be reverted.

File: test/dlc_tests/conftest.py (hunk 1 above)
Comment (tejaschumbalkar) [reply test/dlc_tests/conftest.py:195]: Is the `90Gb` size requirement for neuron as this is a temp change. Once reverted, the instance will have `50Gb` by default

File: test/test_utils/__init__.py [hunk 6]
```diff
@@ -32,11 +32,13 @@
 UBUNTU_18_BASE_DLAMI_US_EAST_1 = "ami-0e3e654c8b1971d8d"
 PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_EAST_1 = "ami-0673bb31cc62485dd"
 PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2 = "ami-02d9a47bc61a31d43"
+NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2 = "ami-0b5d270a84e753c18"
```
Comment (tejaschumbalkar) [inline test/test_utils/__init__.py:35]: Is the AMI public?

Comment (saimidu) [reply test/test_utils/__init__.py:35]: Yup, this is the public Ubuntu 18.04 Base DLAMI.

Also, the 90 GB size requirement is for Neuron, yes.

**Review by tejaschumbalkar — APPROVED** [review]

**Pull request merged**